common/util.go

```diff
@@ -342,10 +342,16 @@ func IsServiceHandlerRetryableError(err error) bool {
 		return false
 	}
 
-	switch err.(type) {
+	switch t := err.(type) {
```

Comment:
the usual way to write this is
```
  switch err := err.(type) {
```
i.e. reuse the name for the type-asserted name